src/Raven.Server/Documents/Replication/ReplicationDocumentSender.cs

```diff
@@ -535,6 +538,13 @@ private bool AddReplicationItemToBatch(ReplicationBatchItem item, OutgoingReplic
 
         private bool ShouldSkip(ReplicationBatchItem item, OutgoingReplicationStatsScope stats, SkippedReplicationItemsInfo skippedReplicationItemsInfo)
         {
+            if (_allowedPathsValidator?.ShouldAllow(item) == false)
+            {
+                stats.RecordArtificialDocumentSkip();
+                skippedReplicationItemsInfo.Update(item);
```

Comment:
Need to write a correct log message, don't want this to mislead us.

Reply:
done